docs/config.schema.json

```diff
@@ -0,0 +1,816 @@
+          "properties": {
+            "bcrypt": {
+              "type": "object",
+              "description": "Configures the BCrypt hashing algorithm used for hashing Client Secrets.",
+              "properties": {
+                "cost": {
+                  "type": "integer",
+                  "description": "Sets the BCrypt cost. The higher the value, the more CPU time is being used to generate hashes.",
+                  "default": 10,
+                  "minimum": 4
+                }
+              }
+            }
+          }
+        },
+        "pkce": {
+          "type": "object",
+          "properties": {
+            "enforced": {
+              "type": "boolean",
+              "description": "Sets whether PKCE should be enforced for all clients.",
+              "examples": [
+                true
+              ]
+            }
+          }
+        }
+      }
+    },
+    "secrets": {
+      "type": "object",
+      "description": "The secrets section configures secrets used for encryption and signing of several systems. All secrets can be rotated, for more information on this topic go to: https://www.ory.sh/docs/hydra/advanced#rotation-of-hmac-token-signing-and-database-and-cookie-encryption-keys",
+      "properties": {
+        "system": {
+          "type": "array",
+          "description": "The system secret must be at least 16 characters long. If none is provided, one will be generated. They key is used to encrypt sensitive data using AES-GCM (256 bit) and validate HMAC signatures. The first item in the list is used for signing and encryption. The whole list is used for verifying signatures and decryption.",
+          "items": {
+            "type": "string",
+            "minLength": 16
+          },
+          "minItems": 1,
+          "examples": [
+            [
+              "this-is-the-primary-secret",
+              "this-is-an-old-secret",
+              "this-is-another-old-secret"
+            ]
+          ]
+        },
+        "cookie": {
+          "type": "array",
+          "description": "A secret that is used to encrypt cookie sessions. Defaults to secrets.system. It is recommended to use\ta separate secret in production. The first item in the list is used for signing and encryption. The whole list is used for verifying signatures and decryption.",
+          "items": {
+            "type": "string",
+            "minLength": 16
+          },
+          "examples": [
+            [
+              "this-is-the-primary-secret",
+              "this-is-an-old-secret",
+              "this-is-another-old-secret"
+            ]
+          ]
+        }
+      }
+    },
+    "profiling": {
+      "type": "string",
+      "description": "Enables profiling if set. For more details on profiling, head over to: https://blog.golang.org/profiling-go-programs",
+      "enum": [
+        "cpu",
+        "mem",
+        ""
```

Comment:
I think this can be removed as empty is just leaving it out of the definition right?